README.md

```diff
@@ -11,7 +22,13 @@ This tool can be used as a workaround for https://github.com/aspnet/EntityFramew
 
   -n, --namespace    Namespace for the generated configuration classes
 
+  -s, --suffix       Suffix for the generated configuration files
+  
   -B, --no-backup    Don't keep a copy of the original DbContext file
+  
+  -r, --no-replace   Don't replace OnModelCreating event code
+ 
+  -e, --embed-config Embed EntityTypeConfiguration class into entity model file
```

Comment:
Should probably be either `--embed-configs` or `--embed-configurations`. Should also be `classes` (plural).